hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/checker/RequiredFieldsChecker.groovy

```diff
@@ -224,6 +225,14 @@ class RequiredFieldsChecker extends BaseStaticChecker {
   }
 
   @Override
+  void visitJob(TableauJob job) {
+    if (job.workbookName == null) {
```

Comment:
That seems like a good idea. I know I handle it in my job type code, but handling it in the DSL would be more immediate feedback for users. Thanks!